src/lib/config.ts

```diff
@@ -83,6 +91,10 @@ export const config: AppConfig = appConfigSchema.parse({
   spotifyClientSecret: process.env.NEXT_PUBLIC_SPOTIFY_CLIENT_SECRET,
   youtubeClientId: process.env.NEXT_PUBLIC_YOUTUBE_CLIENT_ID,
   youtubeClientSecret: process.env.NEXT_PUBLIC_YOUTUBE_CLIENT_SECRET,
+  loginSuccessUrl: process.env.NEXT_PUBLIC_LOGIN_SUCCESS_URL,
```

Comment:
I know this doesn't follow the current convention, but soon the rest will be fixed. Please use the same in .env as you use here. So this would be `NEXT_PUBLIC_LOGIN_SUCCESS_URL` and similar for the rest.

Reply:
I can't know what you mean, would you confirm again?

Reply:
```suggestion
  NEXT_PUBLIC_LOGIN_SUCCESS_URL: process.env.NEXT_PUBLIC_LOGIN_SUCCESS_URL,
```
And the same for the other 3 below.

The change will need to be made in the zod and type definition above too.

Reply:
I dont agree, because it is original format, 
If you want to change it, you can do it later

Reply:
All other config paras are same format too